src/EventSubscriber/ProxySubscriber.php

```diff
@@ -56,7 +57,11 @@ public function __construct(FetchManagerInterface $manager, LoggerInterface $log
    */
   public function checkFileOrigin(GetResponseEvent $event) {
     $file_dir = $this->manager->filePublicPath();
-    if (strpos(request_path(), $file_dir) !== 0) {
+    $uri = $event->getRequest()->getPathInfo();
+
+    $uri = Unicode::substr($uri, 1, Unicode::strlen($uri) - 1);
```

Comment:
The second parameter of substr is optional. No need to calculate the end of the string.